lib/css/stacks-static.less

```diff
@@ -1,22 +1,11 @@
-//
-//  STACK OVERFLOW
-//  STATIC STACK ELEMENTS
-//
-//  This CSS comes from Stacks, our CSS & Pattern library for rapidly building
-//  Stack Overflow. For documentation of all these classes and how to contribute,
-//  visit https://stackoverflow.design/
-//
-//  This is where all the magic happens.
-//
-//  ============================================================================
-//  $   STATIC ELEMENTS
-//      The following items are elements which we DO NOT allow communities
-//      to modify via variables.
-//  ----------------------------------------------------------------------------
+// stacks-static.less contains styles which we DO NOT allow communities to modify via variables
```

Comment:
One note to make here is that when we share Stacks from SO to other projects/sites (e.g. StackExchange.com) _only_ the static styles are included (for legacy theming reasons that don't matter any longer now that we're using css variables after we remove `stacks-dynamic.less`).

No action necessary.